Islandora-Devops/islandora_starter_theme
# Add git ignore and corrected missing node modules

This removes the node_modules directory, I ran the gulp command so it updated the CSS output as well.

## To test this
- cd into the theme
- Switch to this PR
- Run the `npm -i` to build
- Run `drush cr` to clear cache to see.

It should be exactly the same.
